### PR TITLE
chore(playground): add playground and initial playwright tests

### PR DESCRIPTION
**Pull request by mkilpatrick**

### CODE REVIEW

File: playground/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (tatimblin) [inline playground/tsconfig.json:1]: Can this be [inherited](https://www.typescriptlang.org/tsconfig#extends) from a base config up a directory?

**Review by tatimblin — COMMENTED** [review]
The config and starter site look fine. For naming 'playground' sounds like we can freely edit without worrying, but I think that would make tests fail? and 'react' means that there would be other non react test sites.

File: playground/tsconfig.json (hunk 1 above)
Comment (mkilpatrick) [reply playground/tsconfig.json:1]: Probably

**Comment by mkilpatrick** [comment]
> The config and starter site look fine. For naming 'playground' sounds like we can freely edit without worrying, but I think that would make tests fail? and 'react' means that there would be other non react test sites.

Might want to rename to pages-starter-react-locations

**Comment by oshi97** [comment]
Closing this PR since we've merged in a playwright tests in a different PR, will keep the branch around though

**Pull request closed without merging**